src/App.css

```diff
@@ -1,3 +1,4 @@
+@import '~antd/dist/antd.css';
```

Comment:
how did you import this since Hibar still didn't, please can you remove it for now till hiba add it